Makefile

```diff
@@ -35,3 +35,7 @@ gen-php: install-buf guard-GOPATH
 
 gen-ruby: install-buf guard-GOPATH
 	${GOPATH}/bin/buf generate buf.build/open-feature/flagd --template protobuf/buf.gen.ruby.yaml
+
+ajv-validate-flagd-schema:
+	npm -g --no-fund install ajv-cli
```

Comment:
Perhaps it would be better to do something like this:
https://github.com/open-feature/spec/blob/main/Makefile#L28

I'm not sure if the makefile should install an npm packages globally.